templates/workspace_services/guacamole/user_resources/guacamole-dev-vm/terraform/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Do you need this script?

Reply:
I think so? It's a direct copy of the linux-vm where I assume it's needed?